.gitbook/contents/hc/3_write-smart-contract.md

```diff
@@ -0,0 +1,190 @@
+# Write the Smart Contract
+
+Now we can write the smart contract, which will call our previously created off-chain handler. Our contract has two purposes:
+
+1. Intentionally burn ETH gas (in order to test gas estimation logic).
+2. Count how many times the contract has been called based on certain inputs and errors.
```

Comment:
This should be "increment a counter variable" rather than "count how many times"